src/scripts/main.js

```diff
@@ -353,8 +353,21 @@ const people = [
     slug: 'jacobus-bernardus-van-brussel-1736',
   },
 ];
+const table = document.querySelector('table');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach((el) => {
+  el.Age = el.died - el.born;
+  el.Century = Math.ceil(el.died / 100);
+  el.sex = el.sex === 'y' ? 'female' : 'male';
 
-// write your code here
+  const row = `
+      <td>${el.name}</td>
+      <td>${el.sex}</td>
+      <td>${el.born}</td>
+      <td>${el.died}</td>
+      <td>${el.Age}</td>
+      <td>${el.Century}</td>
+  `;
+
+  table.innerHTML += row;
```

Comment:
You should avoid using `innerHTML +=` for appending elements because it leads to high memory consumption. It's better to create elements using document.createElement() and then append them using appendChild().